eva_cttv_pipeline/clinvar_to_evidence_strings.py

```diff
@@ -270,47 +273,47 @@ def skip_record(clinvar_record, cellbase_record, allowed_clinical_significance,
     if clinvar_record.clinical_significance not in allowed_clinical_significance:
         if clinvar_record.nsv is not None:
             counters["n_nsv_skipped_clin_sig"] += 1
-        print("Not in allowed. Clinical significance: %s. Allowed clincal significances: %s." %
-              (clinvar_record.clinical_significance, allowed_clinical_significance))
+        # print("Not in allowed. Clinical significance: %s. Allowed clincal significances: %s." %
+              # (clinvar_record.clinical_significance, allowed_clinical_significance))
```

Comment:
This and the following commented blocks should be removed or logged into the standard error stream.